mlflow/utils/uri.py

```diff
@@ -265,3 +266,51 @@ def is_valid_dbfs_uri(uri):
     except MlflowException:
         db_profile_uri = None
     return not parsed.netloc or db_profile_uri is not None
+
+
+def is_databricks_profile(uri):
+    profile_uri = get_databricks_profile_uri_from_artifact_uri(uri) or mlflow.get_registry_uri()
+    return is_databricks_uri(profile_uri)
+
+
+# Model URI Utils
```

Comment:
It seemed a bit weird to have a utils file inside `mlflow/store/artifact/`.
There are two options:
1) I can nested it under `mlflow/store/artifact/utils`.
2) We do have a model_utils.py in `mlflow.utils`, and it currently contains utils directed towards model flavors. 

I am following option 2. Will that be alright?

Reply:
i think it's ok to have the utils in the artifacts folder since they are scoped as such (examples under tracking). i definitely would not put them in model_utils since that is for the Model (MLmodel) component.

Reply:
(option 1 sounds fine to me)